src/main/resources/jupyter/init-actions.sh

```diff
@@ -14,6 +14,27 @@ function update_apt_get() {
   return 1
 }
 
+function run_docker_compose() {
+  COMPOSE_FILE=$1
+  docker-compose -f $COMPOSE_FILE config
```

Comment:
Can we parameterize the max number of retries and the period in between for easier visibility/configurability?